### DISCUSSION TITLE
delete specific dependencies

### DISCUSSION THREAD

**Discussion by andife** [Festo-se/cyclonedx-editor-validator#39] [Q&A]
Hi,

Is there a way to delete specific dependenciesfrom the list? for example "pip" at python SBOM?

**Comment by italvi** [comment]
Hi @andife,
I'm really sorry about the huge delay in response. Unfortunately, I have not notifications on for discussions and saw your question too late. Is this topic still up-to-date? If yes, could you please further explain your question, i.e. are you aiming to remove information from your SBOM? I would close the discussion for now, but if it is still valid, please reopen it 😁
